src/accordion.ts

```diff
@@ -0,0 +1,104 @@
+import { LitElement, html, css } from "lit";
+import { customElement, state, property } from "lit/decorators.js";
+
+@customElement("aalam-accordion")
+export class AalamAccordion extends LitElement {
+    static override styles = css`
+       
+    `;
+
+    @property()
+    togglesel: string = ".acc-title";
+
+    @property()
+    bodysel: string = ".acc-body";
+
+    @property({ type: Boolean })
+    nocloseall: boolean = false;
+
+    @property({ type: Boolean })
+    nomultiple: boolean = false;
+
+    @property({ type: Boolean })
+    alwaysopen: boolean = false;
+
+    @property({ type: String })
+    activecls: string = "acc-active";
+
+    @state()
+    open_count: number = 0;
+
+    override render() {
+        return html`<slot >
+
+        </slot>`;
+    }
+
+    override firstUpdated() {
+        const items = this.querySelectorAll("div");
```

Comment:
We cant be so sure if the child elements are all div elements. It can be anything.